src/pythonfinder/models/path.py

```diff
@@ -356,33 +405,31 @@ def find_python_version(
 
         if isinstance(major, six.string_types) and not minor and not patch:
             # Only proceed if this is in the format "x.y.z" or similar
-            if major.count(".") > 0 and major[0].isdigit():
+            if major.isdigit() or (major.count(".") > 0 and major[0].isdigit()):
                 version = major.split(".", 2)
-                if len(version) > 3:
-                    major, minor, patch, rest = version
-                elif len(version) == 3:
-                    major, minor, patch = version
+                if isinstance(version, (tuple, list)):
+                    if len(version) > 3:
+                        major, minor, patch, rest = version
+                    elif len(version) == 3:
+                        major, minor, patch = version
+                    elif len(version) == 2:
+                        major, minor = version
+                    else:
+                        major = version
```

Comment:
```suggestion
                        major = version[0]
```

mypy is complaining ```error: Incompatible types in assignment (expression has type "List[str]", variable has type "Union[str, int, None]")```

Reply:
hrm that's a good call, I was debating if I should leave it as a string or as you say pass it as a list, but I'm pretty sure this is right